src/components/Home/Home.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../styles/abstracts/variables";
+
+.Home {
+    display: grid;
+    grid-template-rows: 2fr 3fr;
+    min-height: calc(100vh - 50px);
+    text-align: center;
+}
+
+.Hero {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: $color-gray-bg;
+}
+
+.Header {
+    margin-bottom: 30px;
+}
+
+.StartGame {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.Button {
```

Comment:
how about `StartGameAction`?
later on there can be some other button on this view, then we will be confused, oh no